### PR TITLE
infra: edge zones

### PR DESCRIPTION
**Pull request by shwstppr**
For feature: https://github.com/apache/cloudstack/pull/6840

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/installguide/configuration.rst [hunk 1]
```diff
@@ -521,8 +521,12 @@ Basic Zone Configuration
 Advanced Zone Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. After you select Advanced in the Add Zone wizard and click Next, you
-   will be asked to enter the following details. Then click Next.
+For Advanced zone, you may chose Edge which will allow creating an Edge. If Edge is not selected then wizard will allow creating a Core zone.
```
Comment (DaanHoogland) [inline source/installguide/configuration.rst:524]: can you explain or modify this sentence? this is a checkbox ? maybe you mean 
```suggestion
For Advanced zone, you may chose to select Edge which will allow creating an Edge Zone. If Edge is not selected then wizard will continue creating a Core zone.
```

File: source/installguide/configuration.rst [hunk 2]
```diff
@@ -764,6 +768,56 @@ Advanced Zone Configuration
 #. Click Launch.
 
 
+Edge Zone
+*********
+
+.. note::
+   Support for Edge zone has been added with 4.18.0 and these zones will only support KVM hypervisor
```
Comment (DaanHoogland) [inline source/installguide/configuration.rst:775]:
```suggestion
   Support for Edge zones has been added with 4.18.0 and these zones will only be supported on KVM hypervisors
```

File: source/installguide/configuration.rst [hunk 3]
```diff
@@ -764,6 +768,56 @@ Advanced Zone Configuration
...
+
+Edge zone is a simpler, light-weight zone which may often contain a single hypervisor host. There will be no need for shared storage, public and management physical networks for an Edge zone.
```
Comment (DaanHoogland) [inline source/installguide/configuration.rst:777]:
```suggestion
An Edge Zone is a simpler, light-weight zone which may often contain a single hypervisor host. There will be no need for shared storage, public and management physical networks for an Edge zone.
```

File: source/installguide/configuration.rst [hunk 4]
```diff
@@ -764,6 +768,56 @@ Advanced Zone Configuration
...
+To work with limited compute resources, an Edge zone will not deploy system VMs. This type of zone only support shared and L2 guest networks. For virtual routers of a shared guest network, a direct-download System VM must be added after adding the zone.
```
Comment (DaanHoogland) [inline source/installguide/configuration.rst:778]:
```suggestion
To work with limited compute resources, an Edge zone will not deploy system VMs. This type of zone only supports shared and L2 guest networks. For virtual routers of a shared guest network, a direct-download System VM must be added after adding the zone.
```

**Review by DaanHoogland — COMMENTED** [review]
most are just suggestions, but I was really confused by the opening sentence

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Review by NuxRo — APPROVED** [review]
LGTM

**Comment by alexandremattioli** [comment]
LGTM

**Pull request merged**